### PR TITLE
Feature/implementing drf

### PR DESCRIPTION
**Pull request by slevvvvin**

### CODE REVIEW

File: myDjangoBlog/settings.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['0.0.0.0', 'localhost']
```
Comment (TinaArts) [inline myDjangoBlog/settings.py]: Allowed host shout be in settings for dev env

File: blog/api/views.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    serializer_class = CategoryListSerializer
+
+
+class CategoryDetailAPIView(generics.RetrieveUpdateDestroyAPIView):
+
+    queryset = Category.objects.all()
+    serializer_class = CategoryListSerializer
+
+    @staticmethod
+    def get(request, pk):
```
Comment (TinaArts) [inline blog/api/views.py]: This logic should be in post. 'get' method for categories means that you want to get info about category

**Review by TinaArts — CHANGES_REQUESTED** [review]

File: myDjangoBlog/settings.py (hunk 1 above)
Comment (slevvvvin) [reply myDjangoBlog/settings.py]: done

File: blog/api/views.py (hunk 2 above)
Comment (slevvvvin) [reply blog/api/views.py]: done

**Review by TinaArts — APPROVED** [review]

**Pull request merged**